Add Arctic Wolf Networks blog post (#473)
trinodb/trino.io: _posts/2023-07-05-trino-fest-2023-arcticwolf.md

---
layout: post
title: "AWS Athena (Trino) in the cybersecurity space"
author: "Anas Shakra, Ryan Duan"
excerpt_separator: <!--more-->
image: /assets/blog/trino-fest-2023/ArcticWolf.png
show_pagenav: false
---

AWS Athena has become a new and efficient service for companies to query their
data using Trino. AWS Athena is a serverless, interactive analytics service

> built on open-source frameworks, supporting open-table and file formats and

Reviewer: ```suggestion
built on open-source frameworks, supporting open table and file formats and
```

provides a simplified, flexible way to analyze petabytes of data where it lives.
Senior software developer Anas Shakra from Arctic Wolf Networks gave a talk at
[Trino Fest 2023]({% post_url 2023-06-20-trino-fest-2023-recap %}) detailing
their switch to AWS Athena and how "queries that took hours with old solution

> now take around a minute today." Tune in to the talk or you can read the recap below!

Reviewer: ```suggestion
now take around a minute today". Tune in to the talk or you can read the recap.
```

<!--more-->

{% youtube WCuJaW7zC8k %}

## Recap

> Generally, data access use-cases fall under three categories: investigations,

Reviewer: Add a sentence about what Arctic Wolf does before this paragraph.

Reviewer: Or even better add it to the first paragraph of this blog post about the talk ..

compliance, and customer self-serve platform. The process of preparing the data
follows an established pattern of starting with datastore, performing an
operation to filter or transform the data, and then outputting the data in some

> form of format like a CSV or JSON depending on the client needs. However, their

Reviewer: ```suggestion
format like a CSV or JSON, depending on the client needs. However, their
```

legacy service was unable to match the growing service demand and
had four main problems:

* Optimized for breadth over depth
* Struggles to handle growing service demand
* Proprietary query language
* Complicated design

> This compelled Anas' team to find a different and improved service: AWS Athena.

Reviewer: Maybe say "Trino as provided by AWS Athena"

They had four main objectives for the new service: defined access patterns,
performant at scale, user-friendly, and deterministic pricing. AWS Athena
satisfied these objectives, while also providing numerous benefits such as using

> a powerful query engine known as Trino, being purposefully built for large

Reviewer: reword keeping in mind that readers her already know Trino or are on the actual website and can look up more here

datasets, using SQL syntax, and having a clear pricing structure. However, with

> these benefits come some drawbacks. These included subjecting to quota limits,

Reviewer: Clarify that those drawbacks are from Athena. .. not Trino.

> having suboptimal file sizes for their datastore, and unable to control access

Reviewer: filesize is problem of their system

> in their preferred defined patterns. Anas addresses this by using log queries

Reviewer: not sure what that means "control access in their preferred defined patterns" ?

that resolves these three main impediments. These changes drastically improved their
performance and they still have major improvements yet to resolve that might

> further enhance their performance.

Reviewer: Also maybe add that they are considering moving to Trino itself and run it on their own ..

---
layout: post
title: "AWS Athena (Trino) in the cybersecurity space"
author: "Anas Shakra, Ryan Duan"
excerpt_separator: <!--more-->
image: /assets/blog/trino-fest-2023/ArcticWolf.png
show_pagenav: false
---

AWS Athena has become a new and efficient service for companies to query their
data using Trino. AWS Athena is a serverless, interactive analytics service
built on open-source frameworks, supporting open table and file formats and
providing a simplified, flexible way to analyze petabytes of data where it lives.

> Arctic Wolf Networks, a cybersecurity company that provides security monitoring

Reviewer 2: I would re-order this paragraph. Start by introducing who Arctic Wolf is (it's their talk), then explain what Athena is after mentioning that htey use it. You should also be more explicit about the fact that Athena runs on Trino.

to cyber threats, is one of these companies that have recently switched to using
AWS Athena. Senior software developer Anas Shakra from Arctic Wolf Networks gave
a talk at [Trino Fest 2023]({% post_url 2023-06-20-trino-fest-2023-recap %})
detailing their switch to AWS Athena and how "queries that took hours with old
solution now take around a minute today". Tune in to the talk or you can read
the recap!

<!--more-->

{% youtube WCuJaW7zC8k %}

## Recap

> Generally, data access use-cases fall under three categories: investigations,

Reviewer 2: ```suggestion
At Arctic Wolf, data access use-cases fall under three categories: investigations,
```
This isn't general, this is specific to what they do. 😛

Reviewer: ;-)

---
layout: post
title: "AWS Athena (Trino) in the cybersecurity space"
author: "Anas Shakra, Ryan Duan"
excerpt_separator: <!--more-->
image: /assets/blog/trino-fest-2023/ArcticWolf.png
show_pagenav: false
---

AWS Athena has become a new and efficient service for companies to query their
data using Trino. AWS Athena is a serverless, interactive analytics service
built on open-source frameworks, supporting open table and file formats and

> provides a simplified, flexible way to analyze petabytes of data where it lives.

Reviewer: ```suggestion
providing a simplified, flexible way to analyze petabytes of data where it lives.
```

Arctic Wolf Networks, a cybersecurity company that provides security monitoring
to cyber threats, is one of these companies that have recently switched to using
AWS Athena. Senior software developer Anas Shakra from Arctic Wolf Networks gave
a talk at [Trino Fest 2023]({% post_url 2023-06-20-trino-fest-2023-recap %})
detailing their switch to AWS Athena and how "queries that took hours with old
solution now take around a minute today". Tune in to the talk or you can read
the recap!

<!--more-->

{% youtube WCuJaW7zC8k %}

## Recap

Generally, data access use-cases fall under three categories: investigations,
compliance, and customer self-serve platform. The process of preparing the data
follows an established pattern of starting with datastore, performing an
operation to filter or transform the data, and then outputting the data in some

> format like a CSV or JSON, depending on the client needs. However, their legacy

Reviewer: ```suggestion
format like a CSV or JSON, depending on the client needs. Arctic Wolf's custom legacy
```

service was unable to match the growing service demand and had four main
problems:

* Optimized for breadth over depth
* Struggles to handle growing service demand
* Proprietary query language
* Complicated design

This compelled Anas' team to find a different and improved service: Trino as
provided by AWS Athena.

They had four main objectives for the new service: defined access patterns,
performant at scale, user-friendly, and deterministic pricing. AWS Athena
satisfied these objectives, while also providing numerous benefits such as using
a powerful query engine, being purposefully built for large datasets, using SQL
syntax, and having a clear pricing structure. However, with these benefits come

> some drawbacks for Athena. These included subjecting to quota limits, having

Reviewer: ```suggestion
some drawbacks for Athena. These includes being subject to quota limits, having
```

> suboptimal file sizes for their system, and unable to control access to internal

Reviewer: ```suggestion
suboptimal file sizes for their system, and being unable to control access sufficiently.
```

> and external clients. Anas addresses this by using log queries that resolves

Reviewer: ```suggestion
nas addresses this by using log queries that resolves
```

> these three main impediments. These changes drastically improved their further

Reviewer: Get rid of the sentence 

These changes drastically improved their further...

> enhance their performance. In addition, they are considering switching

Reviewer: ```suggestion
As next step Anas is considering switching to a self-managed Trino deployment for more control with the same performance gains.
```
